libgpopt/include/gpopt/translate/CTranslatorExprToDXLUtils.h

```diff
@@ -585,6 +585,10 @@ namespace gpopt
 			// extract the column ids of the ident from project list
 			static
 			void ExtractIdentColIds(CDXLNode *dxlnode, CBitSet *pbs);
+
+			// is this Filter node direct dispatchable?
+			static
+			BOOL FDirectDispatchFilter (CExpression *pexprFilter);
```

Comment:
```suggestion
			BOOL FDirectDispatchableFilter (CExpression *pexprFilter);

```

Can we be consistent about using `DirectDispatchable` and `DirectDispatch`? It seems like everywhere else we use "dispatchable"